README.md

```diff
@@ -55,14 +65,52 @@ vim .env
 ### Run the worker Steps
 
 1. Run the worker
+
 ```sh
 docker compose up -d
 ```
 
+### Run the worker from source
+
+1. Clone this repo
+
+```sh
+git clone https://github.com/Lagrange-Labs/lgn-coprocessor.git
+cd  lgn-coprocessor
+```
+
+2. Check your Rust toolchain
+
+```sh
+rustup toolchain list
+nightly-2024-12-03... (It is recommended)
+```
+
+2. Update the worker configuration
+
+```sh
+cat lgn-worker/src/config/default.toml
+```
+
+3. Update the system maximum stack size to unlimited
+
+```sh
+ulimit -s 100000000 # Only run the worker on one machine
+ulimit -s unlimited # Run the worker with other services on one machine (not recommended)
+```
+
+4. Build and run the worker
+
+```sh
+RUST_BACKTRACE=1 RUST_LOG=debug cargo run --release --bin lgn-worker -- --config YOUR_CONFIG.toml
+```
```

Comment:
Maybe lets also add a default config, and a link to the public params that work with this version of the worker?

Reply:
sounds good to me. @kaisbaccour , just double checking, is the URL `https://pub-a894572689a54c008859f232868fc67d.r2.dev` the one to make public?